README.md

```diff
@@ -40,141 +40,141 @@ npm run lint-fix # format source files
-| [#218][#218] | [MUST contain API meta information][#218] | :heavy_check_mark: |
-| [#219][#219] | [MUST provide API audience][#219] | - |
-| [#220][#220] | [MUST use most specific HTTP status codes][#220] | - |
-| [#223][#223] | [MUST-SHOULD use functional naming schema][#223] | - |
-| [#224][#224] | [MUST follow naming convention for hostnames][#224] | - |
-| [#225][#225] | [MUST follow naming convention for permissions (scopes)][#225] | - |
-| [#226][#226] | [MUST document implicit filtering][#226] | - |
-| [#227][#227] | [MUST document cachable `GET`, `HEAD`, and `POST` endpoints][#227] | - |
-| [#228][#228] | [MUST use URL-friendly resource identifiers: [a-zA-Z0-9:._\-/]*][#228] | - |
-| [#229][#229] | [SHOULD consider to design `POST` and `PATCH` idempotent][#229] | - |
-| [#230][#230] | [MAY consider to support `Idempotency-Key` header][#230] | - |
-| [#231][#231] | [Should use secondary key for idempotent `POST` design][#231] | - |
-| [#233][#233] | [MUST support `X-Flow-ID`][#233] | - |
-| [#234][#234] | [MUST only use durable and immutable remote references][#234] | - |
-| [#235][#235] | [SHOULD name date/time properties with `_at` suffix][#235] | - |
-| [#236][#236] | [SHOULD design simple query languages using query parameters][#236] | - |
-| [#237][#237] | [SHOULD design complex query languages using JSON][#237] | - |
-| [#238][#238] | [SHOULD use standardized property formats][#238] | - |
-| [#239][#239] | [SHOULD encode embedded binary data in `base64url`][#239] | - |
-| [#240][#240] | [SHOULD declare enum values using UPPER_SNAKE_CASE format][#240] | :heavy_check_mark: |
-| [#241][#241] | [MAY expose compound keys as resource identifiers][#241] | - |
+| id | title |supported | prio (A, B, C)|
+| --- | --- | --- | --- |
+| [#100][#100] | [MUST follow API first principle][#100] | - | B |
+| [#101][#101] | [MUST provide API specification using OpenAPI][#101] | - | A |
+| [#102][#102] | [SHOULD provide API user manual][#102] | - | C |
+| [#103][#103] | [MUST write APIs using U.S. English][#103] | - | A |
+| [#104][#104] | [MUST secure endpoints with OAuth 2.0][#104] | - | B |
+| [#105][#105] | [MUST define and assign permissions (scopes)][#105] | - | B |
+| [#106][#106] | [MUST not break backward compatibility][#106] | - | B |
+| [#107][#107] | [SHOULD prefer compatible extensions][#107] | - | ? |
+| [#108][#108] | [MUST prepare clients accept compatible API extensions][#108] | - | C |
+| [#109][#109] | [SHOULD design APIs conservatively][#109] | - | ? |
+| [#110][#110] | [MUST always return JSON objects as top-level data structures][#110] | :heavy_check_mark: | A |
+| [#111][#111] | [MUST treat Open API specification as open for extension by default][#111] | - | - |
+| [#112][#112] | [SHOULD used open-ended list of values (`x-extensible-enum`) for enumerations][#112] | :heavy_check_mark: | - |
+| [#113][#113] | [SHOULD avoid versioning][#113] | - | B |
+| [#114][#114] | [MUST use media type versioning][#114] | - | - |
```

Comment:
If #115 is A then what other versioning should we have?

Reply:
[SHOULD avoid versioning [113]](https://opensource.zalando.com/restful-api-guidelines/#113)
```
If changing an API can’t be done in a compatible way, then proceed in one of these three ways:
 - create a new resource (variant) in addition to the old resource variant
 - create a new service endpoint — i.e. a new application with a new API (with a new domain name)
 - create a new API version supported in parallel with the old API by the same microservice
```
The third option is then done via headers: [MUST use media type versioning [114]](https://opensource.zalando.com/restful-api-guidelines/#114)